File: LeetCode/1688.Count_of_Matches/main.py
#!/usr/bin/python
from solution import Solution

# Unit Test

solutionObject = Solution()
if (solutionObject.numberOfMatches(14) == 13):
    print("PASS")
else:
    print("FAILED")
if (solutionObject.numberOfMatches(7) == 6):
    print("PASS")
else:
    print("FAILED")

File: LeetCode/1688.Count_of_Matches/main.cpp
#include "solution.h"

// Unit Test

int main(){
    Solution s;
    if (s.numberOfMatches(7) == 6) { cout << "PASS" << endl;}
    else { cout << "Test Failed" << endl; }
    if (s.numberOfMatches(14) == 13) { cout << "PASS" << endl;}
    else { cout << "Test Failed" << endl; }
    return 0;
}